src/components/FormatedDate.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+import React, {
+    Component
+} from 'react';
+
+class FormatedDate extends Component {
+  render() {
+    const date =new Date(this.props.dateString);
+    const dateString = date.toLocaleDateString();
+    return (
+      <span>{dateString}</span>
+    );
+  }
+}
+
+FormatedDate.propTypes = {
+  dateString: PropTypes.string.isRequired
+}
```

Comment:
While its good practice to make sure that all props for all components are defined with Proptypes, you've done enough on these that I think you grasp the concept!